tox.ini

```diff
@@ -33,3 +35,4 @@ deps =
     docs: setuptools_scm
     django21: django>=2.1,<2.2
     django22: django>=2.2,<2.3
+    django30: django>=3.0a1,<3.1
```

Comment:
Use ```django~=3.0```  instead.

Reply:
It seems that this is not correct until version 3.0.0 is released. Let him stay that way for now.

Reply:
`django~=3.0` won't be correct after the release of 3.0.0 as well.  It's equivalent to `>=3.0, ==3.*`.  https://www.python.org/dev/peps/pep-0440/#compatible-release

`django~=3.0.0` is what you were looking for. :)